src/components/shows/showviewer.js

```diff
@@ -0,0 +1,194 @@
+
+class ShowViewer extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      showSelected: false,
+      year: "2019"
+    }
+    this.selectShow = this.selectShow.bind(this)
+  }
+
+  selectShow(input) {
+    this.setState({ year: input, showSelected: true })
+  }
+
+  render() {
+    const landSection = this.state.showSelected ? styles.hide : styles.show
+    const showSection = this.state.showSelected ? styles.show : styles.hide
+
+    let programmeView = styles.show
+    let programmeLink = data[this.state.year]["programme"]
+    if (data[this.state.year]["programme"] === "null") {
+      programmeView = styles.hide
+      programmeLink = data[this.state.year]["programme"]
+    }
+
+    let castPicView = styles.show
+    let castPicture = data[this.state.year]["cast-picture"]
+    let joinusStyle = styles.castText
+    if (data[this.state.year]["cast-picture"] === "null") {
+      castPicView = styles.hide
+      castPicture = data[this.state.year]["cast-picture"]
+      joinusStyle = styles.castText2
+    }
+
+    // =====================================================
+    // TODO(alex-ruddell) Figure out why this is crashing...
+    //                      Small fix done but not perfect.
+    // =====================================================
+
+    let next = parseInt(this.state.year) + 1;
+    let prev = parseInt(this.state.year) - 1;
+    let nextText, prevText, nextYear, prevYear, nextShowDefined, prevShowDefined;
+    if (String(next) === "2020") {
+        nextShowDefined = false;
+        nextYear = "2019"; // Won't be displayed, but this ensures it doesn't crash.
+        nextText = "VIEW ALL SHOWS";
+    } else {
+        nextShowDefined = true;
+        nextYear = next;
+        if (typeof next !== undefined) {
+          nextText = "< " + data[next]['title'];
+        }
+        if (nextText.length > 37) {
+          nextText = nextText.substr(0, 37) + "..."; // Handles case where string is too long.
+        }
+    }
+    if (String(prev) === "2007") {
+        prevShowDefined = false;
+        prevYear = "2019"; // Won't be displayed, but this ensures it doesn't crash.
+        prevText = "VIEW ALL SHOWS";
+    } else {
+        prevShowDefined = true;
+        prevYear = prev;
+        if (typeof prev !== undefined) {
+          prevText = data[prev]['title'] + " >";
+        }
+        if (prevText.length > 37) {
+            prevText = prevText.substr(0, 35) + "... >"; // Handles case where string is too long.
+        }
+    }
+
+    return (
+      <>
+        <HeroBanner year={this.state.year} defined={this.state.showSelected} />
+        <section className={styles.section}>
+          <section className={styles.container}>
+            <section className={showSection}>
+              {/* To display specific show information*/}
+              <div className={styles.topButtons}>
+                
+                <div className={styles.fwrdButton}>
```

Comment:
These buttons are not accessible to people who cannot use a mouse. These should be wrapped in a `<button>` element so that the browser will take care of this for you. (This also brings additional benefits like applying the proper "hand" cursor without needing to set it in CSS).